### DISCUSSION TITLE
Repo structures

### DISCUSSION THREAD

**Discussion by bardic** [nova-scotia-commie/tiktok#3] [Ideas]
Hey hey all
what are some thoughts on ways to layout information here? Should we use just discussions? Or have most information added to the repo via PRs? 

I think the PR method could be fun lol

If we went that approach we could commit mostly images and markdown. 

Atm we have a LICENSE and a README

 I suggest to start we have 3ish root folders 

 ```
 .
 - tools
   - {lang}
     - {file per tool - file has name, link, description, use case info, etc}
 - shares
   - {sub folder if logical}
     - {file per tool - file has name, link, description, use case info, etc}
 - tutorials
   - {sub folder if logical }
     - {file per tool - file has name, link, description, use case info, etc}
```

OR! 

We do two or three massive markdowns with a ToC and all that jazz?

**Comment by DevNDanger** [comment]
I like the first one
